src/cmd/compile/internal/gc/ssa.go

```diff
@@ -1010,17 +1010,52 @@ func (s *state) rawLoad(t *types.Type, src *ssa.Value) *ssa.Value {
 }
 
 func (s *state) store(t *types.Type, dst, val *ssa.Value) {
+	if isTroubleType(t) {
+		fmt.Printf("storing padded 4-4-8 type: %v\n", t)
```

Comment:
this should be stderr (and below) so redo doesn't consider it part of the file output probably